### PR TITLE
[Docs] Document CI checks, purposes, points of contact

### PR DESCRIPTION
**Pull request by electronjoe**
Needs much team-wide review assistance to fill in ?'s and unknown POCs.

I could dig and fill in more by evaluation of the CircleCI configs etc - but this might be more time effective if PoC / team members simply annotate this PR with corrective asks.

Signed-off-by: Scott Harrison Moeller <smoeller@fb.com>

### CODE REVIEW

File: docs/readmes/contributing/contribute_ci_checks.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+| ci/circleci: c-cpp-codecov | AGW c/c++ unit tests and push coverage to codecov.io | electronjoe (GH), Scott Moeller (slack) |
+| ci/circleci: cloud-test | ? |  |
+| ci/circleci: cloud_lint | ? |  |
+| ci/circleci: cwag-precommit | ? |  |
+| ci/circleci: cwf-operator-build | ? |  |
+| ci/circleci: cwf-operator-precommit | ? |  |
+| ci/circleci: eslint | ? |  |
+| ci/circleci: feg-build | ? |  |
+| ci/circleci: feg-lint | ? |  |
+| ci/circleci: feg-precommit | ? |  |
```
Comment (uri200) [inline docs/readmes/contributing/contribute_ci_checks.md]: for the feg:
```
| ci/circleci: feg-build | Builds Docker images for FEG | themarwhal (GH), Marie Bremner (slack); mpgermano(GH), Michael Germano (slack); uri200 (GH), Oriol Batalla (slack); @emakeev (GH), Evgeniy Makeev (slack) |
| ci/circleci: feg-lint | FEG Go linter and code coverage | themarwhal (GH), Marie Bremner (slack); mpgermano(GH), Michael Germano (slack); uri200 (GH), Oriol Batalla (slack); @emakeev (GH), Evgeniy Makeev (slack) |
| ci/circleci: feg-precommit | Runs FEG unittest | themarwhal (GH), Marie Bremner (slack); mpgermano(GH), Michael Germano (slack); uri200 (GH), Oriol Batalla (slack); @emakeev (GH), Evgeniy Makeev (slack) |
```

File: docs/readmes/contributing/contribute_ci_checks.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+| Jenkins | ? |  |
+| Jenkins CWAG Libvirt | CWAG Docker Compose based integration test? |  ?themarwhal (GH), Marie Bremner (slack)? |
+| Jenkins LTE | A lighter weight and low flake subset of LTE tests including...? |  |
+| Code scanning results / CodeQL | How does this differ from the other CodeQL / Analyzer stages? |  |
+| Magma CI autolabel | ? |  |
+| Magma-OAI-Jenkins | OAI's MME integration test run on OAI Jenkins | rdefosse (GH), Raphael Defosseux (slack) |
+| ci/circleci: c-cpp-codecov | AGW c/c++ unit tests and push coverage to codecov.io | electronjoe (GH), Scott Moeller (slack) |
+| ci/circleci: cloud-test | ? |  |
+| ci/circleci: cloud_lint | ? |  |
+| ci/circleci: cwag-precommit | ? |  |
```
Comment (themarwhal) [inline docs/readmes/contributing/contribute_ci_checks.md:30]: themarwhal (GH), Marie Bremner (slack)

File: docs/readmes/contributing/contribute_ci_checks.md (hunk 1 above)
Comment (electronjoe) [reply docs/readmes/contributing/contribute_ci_checks.md]: Thanks for the additions @uri200!

File: docs/readmes/contributing/contribute_ci_checks.md (hunk 2 above)
Comment (themarwhal) [reply docs/readmes/contributing/contribute_ci_checks.md:30]: ^ for `cwag-precommit`. This runs unit tests in `cwf/gateway` and checks that all integration tests build.

File: docs/readmes/contributing/contribute_ci_checks.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+| ci/circleci: cwf-operator-build | ? |  |
+| ci/circleci: cwf-operator-precommit | ? |  |
+| ci/circleci: eslint | ? |  |
+| ci/circleci: feg-build | Builds Docker images for FEG | themarwhal (GH), Marie Bremner (slack); mpgermano(GH), Michael Germano (slack); uri200 (GH), Oriol Batalla (slack); @emakeev (GH), Evgeniy Makeev (slack) |
+| ci/circleci: feg-lint | FEG Go linter and code coverage | themarwhal (GH), Marie Bremner (slack); mpgermano(GH), Michael Germano (slack); uri200 (GH), Oriol Batalla (slack); @emakeev (GH), Evgeniy Makeev (slack) |
+| ci/circleci: feg-precommit | Runs FEG unittest | themarwhal (GH), Marie Bremner (slack); mpgermano(GH), Michael Germano (slack); uri200 (GH), Oriol Batalla (slack); @emakeev (GH), Evgeniy Makeev (slack) |
+| ci/circleci: insync-checkin | ? |  |
+| ci/circleci: lte-test | ? Is this dupe with Jenkins LTE?... |  |
+| ci/circleci: mme_test | Validate that MME related unit tests build and pass | themarwhal (GH), Marie Bremner (slack) |
+| ci/circleci: nms-build | Validate that the NMS builds | |
```
Comment (themarwhal) [inline docs/readmes/contributing/contribute_ci_checks.md]: We can assign all nms tests to @karthiksubraveti and @andreilee ?

Comment (electronjoe) [reply docs/readmes/contributing/contribute_ci_checks.md]: Done!

File: docs/readmes/contributing/contribute_ci_checks.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+| ci/circleci: cloud_lint | ? |  |
+| ci/circleci: cwag-precommit | ? |  |
+| ci/circleci: cwf-operator-build | ? |  |
+| ci/circleci: cwf-operator-precommit | ? |  |
+| ci/circleci: eslint | ? |  |
+| ci/circleci: feg-build | Builds Docker images for FEG | themarwhal (GH), Marie Bremner (slack); mpgermano(GH), Michael Germano (slack); uri200 (GH), Oriol Batalla (slack); emakeev (GH), Evgeniy Makeev (slack) |
+| ci/circleci: feg-lint | FEG Go linter and code coverage | themarwhal (GH), Marie Bremner (slack); mpgermano(GH), Michael Germano (slack); uri200 (GH), Oriol Batalla (slack); emakeev (GH), Evgeniy Makeev (slack) |
+| ci/circleci: feg-precommit | Runs FEG unittest | themarwhal (GH), Marie Bremner (slack); mpgermano(GH), Michael Germano (slack); uri200 (GH), Oriol Batalla (slack); emakeev (GH), Evgeniy Makeev (slack) |
+| ci/circleci: insync-checkin | ? |  |
+| ci/circleci: lte-test | ? Is this dupe with Jenkins LTE?... |  |
```
Comment (themarwhal) [inline docs/readmes/contributing/contribute_ci_checks.md:38]: lte-test runs non-SUDO python unit tests. Specified in these files: [(orc8r/lte)/gateway/python/defs.mk](https://sourcegraph.com/search?q=repo:%5Egithub%5C.com/magma/magma%24+python/defs.mk+type:path&patternType=literal)

And these unit tests are also run as part of the Jenkins LTE test

File: docs/readmes/contributing/contribute_ci_checks.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+| GCC Warnings & Errors / build_oai | Annotate Pull Requests with any GCC Warnings or Errors for MME target | electronjoe (GH), Scott Moeller (slack) |
+| GCC Warnings & Errors / build_session_manager | Annotate Pull Requests with any GCC Warnings or Errors for session_manager target | electronjoe (GH), Scott Moeller (slack) |
+| Jenkins | ? |  |
+| Jenkins CWAG Libvirt | CWAG Docker Compose based integration test? |  themarwhal (GH), Marie Bremner (slack) |
+| Jenkins LTE | A lighter weight and low flake subset of LTE tests including...? |  |
+| Code scanning results / CodeQL | How does this differ from the other CodeQL / Analyzer stages? |  |
+| Magma CI autolabel | ? |  |
+| Magma-OAI-Jenkins | OAI's MME integration test run on OAI Jenkins | rdefosse (GH), Raphael Defosseux (slack) |
+| ci/circleci: c-cpp-codecov | AGW c/c++ unit tests and push coverage to codecov.io | electronjoe (GH), Scott Moeller (slack) |
+| ci/circleci: cloud-test | ? |  |
```
Comment (themarwhal) [inline docs/readmes/contributing/contribute_ci_checks.md:28]: cloud-test/lint is probably hunter / karthik?

**Review by themarwhal — APPROVED** [review]

**Pull request merged**